### PR TITLE
Added "when cmoc" for people that spam it.

### PR DESCRIPTION
**Pull request by WiiNoMa**
ug

### CODE REVIEW

**Comment by TheMegaEagle** [comment]
thanks but will be merged when 1.4.0 is done to assure theres no conflicts while updating

**Comment by WiiNoMa** [comment]
kthx

File: Settings/error-db.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
     "109139":{"description":"[WC24] WiiConnect24 is disabled.", "fix":"Go to Wii Settings > Internet Settings, and enable it."},
     "231000":{"description":"[WC24] Data could not be sent. Check your internet settings.", "fix":"If you're getting this on the Everybody Votes Channel, try restarting or if you get it again, repatch."}, 
     "FORE000006":{"description":"[WC24] Invalid data sent.", "fix":"Try unscrewing your battery cover on your Wii and put it back in a few hours later."},
-    "NEWS0000006":{"description":"[WC24] Invalid data sent.", "fix":"This means there too many news articles the Wii can support. Try again in a few hours or days."}
+    "NEWS0000006":{"description":"[WC24] Invalid data sent.", "fix":"This means there too many news articles the Wii can support. Try again in a few hours or days."},
+    "when cmoc":{"description":"[CMOC..?] Don't spam this kthx.", "fix":"*sigh* Please stop asking, or according to CornierKhan1, "Here's a fun game to play: Every time someone asks when cmoc is coming out, we'll put off the project for another month. :)""}
```
Comment (spotlightishere) [inline Settings/error-db.json]:
```suggestion
    "when cmoc":{"description":"[CMOC..?] Don't spam this kthx.", "fix":"*sigh* Please stop asking, or according to CornierKhan1, \"Here's a fun game to play: Every time someone asks when cmoc is coming out, we'll put off the project for another month. :)\""}
```

This isn't valid JSON syntax by its own. Escape the quotation marks inside instead.

We also ask you bundle mass changes when creating a PR or mentioning in-server.

Comment (WiiNoMa) [reply Settings/error-db.json]: but im here specifically for adding errors, so how is that fair...

Comment (spotlightishere) [reply Settings/error-db.json]: Collecting more errors helps consolidate the amount of notifications people may get. Several have unwatched already over the mass amounts.
Mentioning in #bot-spam on the bot-specific server would help greatly as well.

**Comment by larsenv** [comment]
That description for "when cmoc" was annoying to read

Plus CornierKhan isn't even working on it, shutterbug is.

**Comment by spotlightishere** [comment]
Everyone is.
Maybe just not actively, but everyone. No need to gateway.

This is much more a joke PR at this point and is going nowhere useful.

**Pull request closed without merging**